vocab/security/template.html

```diff
@@ -298,7 +297,7 @@ <h3>Proof Section</h3>
           <p>
             The right side of the Section contains a column of labeled boxes, 
             all styled as Property. The labels, from top to 
-            bottom, are "previousProof", "domain", "challenge", "proofPurpose", 
+            bottom, are "previousProof", "domain", "challenge",
             "nonce", "created", "proofValue". 
             The ellipse labeled as "Proof" is connected to all of these with 
             connecting lines styled as Domain Of. 
```

Comment:
The "styled as" _whatever_ (throughout) should be set off from surrounding text. Could be quote-wrapped —
```suggestion
            connecting lines styled as "Domain Of". 
```

Reply:
or ital —
```suggestion
            connecting lines styled as <i>Domain Of</i>. 
```

Reply:
or something else.